src/Management/src/EndpointBase/Hypermedia/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Steeltoe.Management;
+using Steeltoe.Management.Endpoint.Hypermedia;
+using System;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Add services used by the Hypermedia actuator
+    /// </summary>
+    public static partial class ServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds the services used by the Hypermedia actuator
+        /// </summary>
+        /// <param name="services">Reference to the service collection</param>
+        /// <param name="configuration">Reference to the configuration system</param>
+        /// <returns>A reference to the service collection</returns>
+        public static IServiceCollection AddHypermediaActuatorServices(this IServiceCollection services, IConfiguration configuration)
```

Comment:
This method doesn't appear to be used in code, did `EndpointCore/Hypermedia/EndpointServiceCollectionExtensions` just get missed?